app/graphql/types/mutation_type.rb

```diff
@@ -4,6 +4,7 @@ class MutationType < Types::BaseObject
     field :sign_in, mutation: Mutations::SignIn
 
     field :create_project, mutation: Mutations::CreateProject
+    field :update_comment, mutation: Mutations::UpdateComment
     field :update_project, mutation: Mutations::UpdateProject
```

Comment:
Can you put this separately from the project mutations?
```suggestion
    field :create_project, mutation: Mutations::CreateProject
    field :update_project, mutation: Mutations::UpdateProject
    
    field :update_comment, mutation: Mutations::UpdateComment
```